docs/index.rst

```diff
@@ -5,7 +5,7 @@ pep8's documentation
 
 .. note::
 
-    This package used to be called ``pep8`` but was renamed to ``pycodestyle``
+    This package used to be called `pep8 <https://www.python.org/dev/peps/pep-0008/>`_ but was renamed to `pycodestyle <https://pypi.python.org/pypi/pep8>`_
```

Comment:
Oh right, my apologies, I now understand the issue. I will rectify this immediately.